Panic exit after running generate

Hi folks. We're currently using Prisma with our nodejs project, but our Go services use static queries that may drift in the future. We'd like to use the Prisma schema as a source of truth and integrate with the code generated from this library. However, can't seem to find a way around the panic error I'm receiving after using the same schema used in our node service.

The error I'm receiving: 

```
$ go run github.com/prisma/prisma-client-go generate
Prisma schema loaded from schema.prisma
Error:
                SpatialRefSys spatialRefSysActions
}
 from file client.gotpl: 230242:28: expected ';', found npx (and 91 more errors)
exit status 1



panic: could not run [generate]: exit status 1

goroutine 1 [running]:
main.main()
        /Users/me/Development/go/pkg/mod/github.com/prisma/prisma-client-go@v0.12.2/main.go:29 +0x40c
exit status 2
```

From what I can tell, the table doesn't actually matter. The error will always return the last table in the schema if I comment out things.

Not sure if it's helpful, but there are 63 tables and nearly 1000 lines of code in the schema. This is at the top of the schema if that helps with debugging:

```
generator client {
  output = "db_gen.go"
  package = "db"
  provider      = "go run github.com/prisma/prisma-client-go"
  // binaryTargets = ["native"]
  // provider      = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

Thanks for taking a look!

--- a/generator/generator.go
+++ b/generator/generator.go
@@ -2,6 +2,7 @@ package generator
 
 import (
 	"os"
+	"strings"
 
 	"github.com/steebchen/prisma-client-go/generator/ast/dmmf"
 	"github.com/steebchen/prisma-client-go/generator/ast/transform"
@@ -20,6 +21,10 @@ type Root struct {
 	// BinaryPaths (optional)
 	BinaryPaths BinaryPaths    `json:"binaryPaths"`
 	AST         *transform.AST `json:"ast"`
+}
+
+func (r *Root) EscapedDatamodel() string {
+	return strings.ReplaceAll(r.Datamodel, "`", "'")
 }
 
 func (r *Root) GetEngineType() string {

--- a/generator/templates/client.gotpl
+++ b/generator/templates/client.gotpl
@@ -1,6 +1,6 @@
 {{- /*gotype:github.com/steebchen/prisma-client-go/generator.Root*/ -}}
 
-const schema = `{{ .Datamodel }}`
+const schema = `{{ .EscapedDatamodel }}`
 const schemaConnectionURL = "{{ (index .Datasources 0).URL.Value }}"
 const schemaEnvVarName = "{{ (index .Datasources 0).URL.FromEnvVar }}"
 

--- a/test/projects/basic/schema.prisma
+++ b/test/projects/basic/schema.prisma
@@ -10,10 +10,13 @@ generator db {
   package           = "basic"
 }
 
+/// User
 model User {
+  /// id
   id       String  @id @default(cuid()) @map("_id")
   email    String  @unique
   username String
   name     String?
+  // stuff with `backtick`
   stuff    String?
 }

